Source/WebKit/WebProcess/WebCoreSupport/WebSearchPopupMenu.h

```diff
@@ -31,6 +31,7 @@ class WebSearchPopupMenu : public WebCore::SearchPopupMenu {
     static Ref<WebSearchPopupMenu> create(WebPage*, WebCore::PopupMenuClient*);
 
     WebCore::PopupMenu* popupMenu() override;
+    RefPtr<WebCore::PopupMenu> protectedPopupMenu() { return popupMenu(); }
```

Comment:
I don't think we need this?

Reply:
Sorry, originally it was complaining about these, but I think it was a consequence of the way I wrote it previously. It seems fine without them now!